src/YoutubeIframe.js

```diff
@@ -211,10 +211,20 @@ const YoutubeIframe = (props, ref) => {
     request => {
       try {
         const url = request.mainDocumentURL || request.url;
-        const iosFirstLoad = Platform.OS === 'ios' && url === 'about:blank';
-        const shouldLoad =
-          iosFirstLoad || url.startsWith(baseUrlOverride || DEFAULT_BASE_URL);
-        return shouldLoad;
+        if (Platform.OS === 'ios') {
+          const iosFirstLoad = url === 'about:blank';
+          if (iosFirstLoad) {
+            return true;
+          }
+          const isYouTubeLink = url.startsWith('https://www.youtube.com/');
+          if (isYouTubeLink) {
+            Linking.openURL(url).catch(error => {
+              console.error('Error opening URL:', error);
```

Comment:
I think I'd probably make this a console.warn rather than an error

Reply:
I think you are right. I've updated and pushed.